### PR TITLE
MODULE 1: CAPSTONE PROJECT 1

### PR DESCRIPTION
**Pull request by Ntwali-Josue**
### In this Pull Request: :

- I add the home page which has a link in the menu to the about page. 
- I add the logo in the header that links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- I add the mobile menu has a close (X) button that closes the menu.
- I add the section "Featured speakers" which is created dynamically in JavaScript.
- I use JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### My loom link 

Please watch my video using this [Loom link](https://www.loom.com/share/d496534aa8e94289899f71215e96056a)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+</head>
+<body>
+    <nav class="nav-header">
+      <i class="fa fa-facebook nav-icons" aria-hidden="true"></i>
+      <i class="fa fa-twitter nav-icons" aria-hidden="true"></i>
+      <ul class="nav-header-list">
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
```
Comment (Emmyn5600) [inline about.html:21]: - [ ] The live link demo in the readme file doesn't work.
link1

File: about.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      <h1 class="logo"><a href="/home.html">LOGO</a></h1>
+      <ul class="nav-header-list ">
+        <li>About</li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>Sponsor</li>
+        <li>News</li>
+        <li><button class="btn-header">JO COMPETITIONS</button></li>
+      </ul>
+    </header>
```
Comment (Emmyn5600) [inline about.html:33]: - [ ] On the mobile menu, I can't access the about page, please fix it.
link3

File: about.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      </section>
+
+      <section class="page-logo">
+        <section class="logo-title">
+          <b>JO COMPETITIONS 2021 Logo</b>
+          <hr>
+        </section>
+        <p class="logo-text">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cupiditate accusamus odit consectetur, quasi neque
+          magnam dicta? Doloribus sed mollitia rem nesciunt eaque minima magni quod esse quis. Dignissimos, eius unde?
```
Comment (Emmyn5600) [inline about.html:69]: - [ ] Let us stick to the design to implement some features as requested. Kindly, implement that section as required. 

-----------------------------------------Original------------------------------------

link5
-----------------------------------------Yours------------------------------------

link4

File: js/index.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    playerBio: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Auttenetur eius maxime voluptate id, ut doloribus ullam ea quas recusandae facilis enim.',
+  },
+  {
+    imgURL: '../images/profile-picture.jpg',
+    playerName: 'KARANGWA Ntwali Josue',
+    playerLocation: 'Rwanda, Kigali',
+    playerBio: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Auttenetur eius maxime voluptate id, ut doloribus ullam ea quas recusandae facilis enim.',
+  },
+];
+
```
Comment (Emmyn5600) [inline js/index.js:52]: - [ ] I see you are trying to get image and information from this file, however, it doesn't appear on the screen as required, please refer to the design to fix the issue.

-------------------------------------Original--------------------------
link7
--------------------------------------Yours-----------------------------
link6

File: about.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    </header>
+    <section class="menu-section">
+      <img src="./images/harmburger.svg" class="menu" alt="harmburger">
+      <nav class="nav">
+        <ul>
+          <li><a href="/about.html">About</a></li>
+          <li><a href="/about.html">Program</a></li>
+          <li><a href="/about.html">Join</a></li>
+          <li><a href="/about.html">Sponsor</a></li>
+          <li><a href="/about.html">News</a></li>
```
Comment (Emmyn5600) [inline about.html:42]: - [ ] Your logo at the navbar links has to be a link that should direct us to the home or main page of the project
link8

File: home.html [hunk 6]
```diff
@@ -0,0 +1,158 @@
...
+          </div>
+          <div class="company">
+            <img src="./images/mozilla.png" alt="image" />
+          </div>
+          <div class="company">
+            <img src="./images/mozilla.png" alt="image" />
+          </div>
+          <div class="company">
+            <img src="./images/mozilla.png" alt="image" />
+          </div>
```
Comment (Emmyn5600) [inline home.html:149]: - [ ] kindly, follow the design to implement some changes as well as convincing the client, refer to the design to work on the partner section.

----------------------------------------------Yours-----------------------------------------------
link9

---------------------------------------------Original--------------------------------------------
Check the Original [here](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

File: home.html [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+    <section class="menu-section">
+      <img src="./images/harmburger.svg" class="menu" alt="harmburger">
+      <nav class="nav">
+        <ul>
+          <li><a href="/about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
```
Comment (Emmyn5600) [inline home.html:50]: - [ ] Some links seem to be broken, please fix it.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Ntwali-Josue 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. In addition please follow the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) provided to implement some features.

### To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### Your commits look more professional:heavy_check_mark:
   * #### You have the descriptive PR:heavy_check_mark:
   
Changes required :writing_hand:  

     * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: about.html (hunk 1 above)
Comment (Ntwali-Josue) [reply about.html:21]: Solved.

File: about.html (hunk 2 above)
Comment (Ntwali-Josue) [reply about.html]: Solved.

File: about.html (hunk 3 above)
Comment (Ntwali-Josue) [reply about.html]: Solved.

File: js/index.js (hunk 4 above)
Comment (Ntwali-Josue) [reply js/index.js]: Solved.

File: about.html (hunk 5 above)
Comment (Ntwali-Josue) [reply about.html]: Solved.

File: home.html (hunk 6 above)
Comment (Ntwali-Josue) [reply home.html]: Solved.

File: home.html (hunk 7 above)
Comment (Ntwali-Josue) [reply home.html]: Solved.

**Comment by Ntwali-Josue** [comment]
@Emmyn5600 thanks for the reviews, I worked on all of them and solved them.

**Review by hemant-soni-vst-au4 — APPROVED** [review]
Hi @Ntwali-Josue, 

Great work on making all the requested changes by the previous reviewer. 🚀 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- When we are on the about page in mobile view, the hamburger menu does not have the link to go back to the home page. Let's try to add the home page link when the user is on the about page.

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ntwali-Josue** [comment]
Thanks a lot @hemant-soni-vst-au4 💯

**Pull request merged**